### ISSUE TITLE
Discussion:  provide more solution for each problems

### ISSUE THREAD

**Issue by yuluo-yx** [devops-ws/learn-code#2]
## ideas

- There are many ways to solve a problem. I think we can write down the different solutions to some problems together.

- Write problem solutions in different languages.

**Comment by LinuxSuRen** [comment]
Of course, I practive my coding skills recently, and record the process in this repository. Please feel free to share yours if you are a user of LeetCode too.
